docs/development.rst

```diff
@@ -539,6 +539,99 @@ For further discussion of good and bad practices regarding commits, see:
 .. _How to Get Your Change Into the Linux Kernel: https://www.kernel.org/doc/Documentation/SubmittingPatches
 
 
+.. _`Making a change`:
+
+Making a change
```

Comment:
suggest that we change this to "Creating and submitting a change to pywbem" . This title was so short it was hard to understand.  Same for pywbemtools.